opnode/rollup/sync/start.go

```diff
@@ -1,47 +1,45 @@
 // The sync package is responsible for reconciling L1 and L2.
 //
+// The Ethereum chain is a DAG of blocks with the root block being the genesis block. At any given
+// time, the head (or tip) of the chain can change if an offshoot/branch of the chain has a higher
```

Comment:
nit: its higher total difficulty, not number. see https://github.com/ethereum/go-ethereum/blob/6c3fea0fc9db561aafd9ec017980d0ac87516b87/core/forkchoice.go#L77